spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -232,6 +232,8 @@ public MessageProperties toMessageProperties(AMQP.BasicProperties source, Envelo
 
 	private volatile int defaultTxSize = DEFAULT_TX_SIZE;
 
+	protected volatile boolean defaultDurableSubscription = true;
+
```

Comment:
Sorry - I see it now 

	boolean durable = !anonymousConsumer && accessor.isDurable(this.defaultDurableSubscription);

But we need to make it very clear in the docs.

The problem is if someone is using a `UUID.randomUUID()` for a group (after the last change we imposed on them :smile: )

But spring-cloud-bus has backed out that change so we're ok there.